### PR TITLE
[#12048] Add tests for FeedbackQuestionsDb

### PR DESCRIPTION
**Pull request by marquestye**
Part of #12048

**Outline of Solution**
- Add verifications in `createFeedbackQuestion`
- Add tests for `FeedbackQuestionsDb`

### CODE REVIEW

File: src/test/java/teammates/storage/sqlapi/FeedbackQuestionsDbTest.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    public void testDeleteFeedbackQuestion_success() {
+        FeedbackQuestion feedbackQuestion = getFeedbackQuestion();
+        UUID fqid = feedbackQuestion.getId();
+
+        mockHibernateUtil.when(() -> HibernateUtil.get(FeedbackQuestion.class, fqid)).thenReturn(feedbackQuestion);
+
+        feedbackQuestionsDb.deleteFeedbackQuestion(fqid);
+
+        mockHibernateUtil.verify(() -> HibernateUtil.remove(feedbackQuestion), times(1));
+    }
```
Comment (cedricongjh) [inline src/test/java/teammates/storage/sqlapi/FeedbackQuestionsDbTest.java:112]: let's also add a a test case that ensures that delete is not called if feedbackquestion is null

**Review by cedricongjh — CHANGES_REQUESTED** [review]
besides the one comment, LGTM! do remember to pull from the latest and fix the component tests

File: src/main/java/teammates/ui/webapi/CreateFeedbackQuestionAction.java [hunk 2]
```diff
@@ -82,6 +83,8 @@ public JsonResult execute() throws InvalidHttpRequestBodyException {
             return new JsonResult(new FeedbackQuestionData(feedbackQuestion));
         } catch (InvalidParametersException ex) {
             throw new InvalidHttpRequestBodyException(ex);
+        } catch (EntityAlreadyExistsException e) {
+            throw new InvalidOperationException("Trying to create a feedback question that already exists", e);
```
Comment (kevin9foong) [inline src/main/java/teammates/ui/webapi/CreateFeedbackQuestionAction.java:87]: Could we reuse the message from the EntityAlreadyExistsException? Since the message is quite similar. Open to discussion on this topic

Comment (kevin9foong) [reply src/main/java/teammates/ui/webapi/CreateFeedbackQuestionAction.java:87]: The existing idea is that the business layer (Actions) should throw a business friendly version of the message, but since it is the same, perhaps we could just reuse to avoid defining new literal strings and enforce consistency across Actions

**Comment by kevin9foong** [comment]
Apart from the existing comments, LGTM!

File: src/main/java/teammates/ui/webapi/CreateFeedbackQuestionAction.java (hunk 2 above)
Comment (cedricongjh) [reply src/main/java/teammates/ui/webapi/CreateFeedbackQuestionAction.java:87]: I agree with this, @marquestye, do make the change here before we merge this in, thank you!

**Comment by weiquu** [comment]
@cedricongjh @kevin9foong ping for review

**Review by cedricongjh — APPROVED** [review]
LGTM, thanks!
